ReadMe.md

```diff
@@ -64,7 +65,7 @@ To display | 表示（する） | ひょうじ |
 
 English | Japanese | Kana | Notes
 --------|----------|------|------
-Device | 端末 | たんまつ | Usually smartphones and tablets
+Device, terminal | 端末 | たんまつ | Often used to mean smartphones and tablets, but can also mean command-line
```

Comment:
I honestly would remove the need for any language wrapping around those examples
also the purpose of this document is not to cover all possibilities, but rather so the most frequented ones
so I would change the Notes section to: `smartphones, tables, command-line`